cmd/node/node.go

```diff
@@ -361,21 +384,23 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 
 	// todo : register all protocols
 
-	sgn := hare.NewMockSigning() //todo: shouldn't be any mock code here
-	pub, _ := crypto.NewPublicKey(sgn.Verifier().Bytes())
+	app.edSgn = getEdIdentity()
```

Comment:
check for errors once returned from this call

Reply:
Fixed